Add cephadm_commands placeholder variable

jovial opened the pull request:
Support for this feature was added in #290. This adds a variable and some documentation to raise awareness of its existence.

jovial committed:
Add cephadm_commands placeholder variable

Support for this feature was added in #290. This adds a variable
and some documentation to raise awareness of its existence.

markgoddard commented on doc/source/configuration/cephadm.rst:
```diff
@@ -239,6 +239,21 @@ for Cinder, Cinder backup, Glance, and Nova in Kolla Ansible.
+~~~~~~~~~~~~~
+
+It is possible to run an arbitrary list of commands against the cluster after deployment
+by setting the ``ceph_commands`` variable. ``ceph_commands`` should be a list of commands
```
```suggestion
by setting the ``cephadm_commands`` variable. ``cephadm_commands`` should be a list of commands
```

mnasiadka commented on doc/source/configuration/cephadm.rst:
```diff
@@ -239,6 +239,21 @@ for Cinder, Cinder backup, Glance, and Nova in Kolla Ansible.
+   # A list of commands to pass to cephadm shell -- ceph. See stackhpc.cephadm.commands
+   # for format.
+   cephadm_commands:
+     - "fs new cephfs cephfs_metadata cephfs_data"
```
I would argue that's a good example, changing placement should rather be done using cephadm role, not the commands role.

jovial replied:
That is the example from the docs in the collection. I was going to use this approach to do:

```
ceph config set mgr mgr/prometheus/server_addr 10.0.0.1
```

So it wasn't bound to all interfaces. Is that a better example?

EDIT: Assuming you meant **not** a good example

mnasiadka replied:
Yes, an mgr var is a good example :-) I meant the old one is NOT a good example.

jovial replied:
Thanks, makes sense to not encourage a bad pattern of usage.

jovial committed:
Update doc/source/configuration/cephadm.rst

Co-authored-by: Mark Goddard <mark@stackhpc.com>

jovial committed:
Update example

Updating the example based on feedback from the code review. The previous example applied configuration that would have been better applied using the cephadm role.  We don't want to encourage such usage.
